source/BaselineOfWillow/BaselineOfWillow.class.st

```diff
@@ -63,5 +64,6 @@ BaselineOfWillow >> setUpPackages: spec [
 		package: 'Willow-SUnit' with: [ spec requires: #('Boardwalk-SUnit' 'Willow-Core') ];
 		group: 'Dependent-SUnit-Extensions' with: 'Willow-SUnit';
 		package: 'Willow-Tests' with: [ spec requires: 'Willow-SUnit' ];
-		group: 'Tests' with: 'Willow-Tests'
+		package: 'Willow-Core-Development-Support' with: [ spec requires: 'Seaside3-Development' ];
+		group: 'Tests' with: #('Willow-Tests' 'Willow-Core-Development-Support')
```

Comment:
Why not in the `Tools` group also? If you're using Willow as a dependency don't need the Tests group but having the Tools one is handy for development.